### PR TITLE
blog post for rentrez paper / API keys

### PR DESCRIPTION
**Pull request by dwinter**
Couple of things to note
*  Post refers to version 1.2.1, which was just submitted to CRAN (so should be up by the time this is published)
* frontmatter doesn't include a topic-id yet
* I haven't previewed this locally (but gather this PR will create a preview locally). 

Happy to make any changes that you spot a need for.

### CODE REVIEW

**Comment by sckott** [comment]
thanks @dwinter !

can you fix your yaml?  not sure what exactly error is but it should follow this format:

```
---
slug: yourslug
title: xxx
authors:
- name: Foo Bar
   url: your url
date: 2018-01-16
categories: blog
topicid: xxx
tags:
- ...
---
```

**Comment by dwinter** [comment]
@sckott , well I managed to squeeze quite a few errors into that one little yaml header, but looks like it's compiling now!

**Comment by sckott** [comment]
thx, see  https://deploy-preview-154--ropensci.netlify.com/blog/2018/03/08/rentrez-paper/

**Comment by stefaniebutland** [comment]
Hi @dwinter. Great to see your paper is published, and as promised, you've put together a post.

I have a couple of suggestions. If you can make the edits relatively soon, we could publish this Tues Mar 6. 
- I think the post would benefit from more background. We have such a broad audience that it's important to say what NCBI is or at least what data eutils accesses (link to https://www.ncbi.nlm.nih.gov/books/NBK25501/ for ex). Most of my suggestions relate to fleshing out the post with more info. The first sentence of your R journal paper is ideal so maybe you can paraphrase to introduce the post.
- If you would rather keep a minimal post then this is more suitable as a technote https://ropensci.org/technotes/

- explain - what does the package do?
- Could you take an example from here https://github.com/ropensci/rentrez to add to the post so people will recognize what they can do with it?
- This bit is handy too https://github.com/ropensci/rentrez#getting-information-about-ncbi-databases

- I will add topic id to YAML when we publish
- this phrase missing a word: "...latest release of does make..."
- does "projects" need an apostrophe here? " file issues at the projects repository "

Take home message is lure people in with the power of accessing NCBI's data with `rentrez` because you only get this one chance to advertise it like this with a tweet that will go to ~17,000 followers :-)

Let me know what you think!

**Comment by dwinter** [comment]
Hi @stefaniebutland, 

I could certainly expand this to include some examples and background. I can work on this tomorrow, and ping you when I push an update.

**Comment by stefaniebutland** [comment]
@dwinter Clicking around I noticed this says there is no code of conduct https://github.com/ropensci/rentrez/community but you do have one. @sckott says they are looking for exact spelling:  `CODE_OF_CONDUCT.md` instead of `CONDUCT.md` fyi.

**Comment by stefaniebutland** [comment]
@dwinter The publication date for your post will be 2018-03-20. Can you please submit updates by 2018-03-13?

**Comment by dwinter** [comment]
Hi @stefaniebutland, that should be fine. Will give you head up if I work out I won't be able to get to it before then

**Comment by dwinter** [comment]
Hi @stefaniebutland, all done from me. Let me know if you have any suggested edits/fixes for the post.

**Comment by stefaniebutland** [comment]
Thanks @dwinter. I'll review this asap, before Friday a.m. Currently in all-day meetings

File: content/blog/2018-03-20-rentrez-paper.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    - software
+    - NCBI
+    - genetics
+    - scholarly-literature
+    - data-access
+---
+
+I am happy to say that the latest issue of _The R Journal_ [includes a paper
+describing rentrez](https://journal.r-project.org/archive/2017/RJ-2017-058/index.html), 
+the R package data from the National Center for Biotechnology Information
```
Comment (stefaniebutland) [inline content/blog/2018-03-20-rentrez-paper.md]: do you mean "R package data" or R data package?

File: content/blog/2018-03-20-rentrez-paper.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+![Dot plot: destination of papers describing R packages in 2017](/img/blog-images/2018-03-20-rentrez-paper/journal_dot_plot.png)
+
+So, it looks like _Bioinformatics_, _BMC Bionformatics_ and _Molecular Ecology
+Resources_ are popular destinations for papers describing R packages, but these
+appear in journals all the way across the biological sciences.
+
+The _R Journal_ article describes some more typical uses of `rentrez`, and also
+describes some of decisions that went into the design of the package. If this
+example has whetted your appetite, then please check out the article of the
```
Comment (stefaniebutland) [inline content/blog/2018-03-20-rentrez-paper.md]: - do you mean "or" the package doc?
- maybe link to pkg documentation there

File: content/blog/2018-03-20-rentrez-paper.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    - scholarly-literature
+    - data-access
+---
+
+I am happy to say that the latest issue of _The R Journal_ [includes a paper
+describing rentrez](https://journal.r-project.org/archive/2017/RJ-2017-058/index.html), 
+the R package data from the National Center for Biotechnology Information
+(NCBI).
+
+[The NCBI](https://www.ncbi.nlm.nih.gov/) is one of the most important sources of biological data. The centre
```
Comment (stefaniebutland) [inline content/blog/2018-03-20-rentrez-paper.md]: I like the added intro and example for context!

File: content/blog/2018-03-20-rentrez-paper.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
+---
+slug: "rentrez paper" 
+title: A rentrez paper, and how to use the NCBI's new API keys 
+package_version: 1.2.1
+authors:
+    - name: David Winter
+    - url: https://david-winter.info
```
Comment (sckott) [inline content/blog/2018-03-20-rentrez-paper.md]: @dwinter change the author block to:

```
authors:
    - name: David Winter
      url: https://david-winter.info
```

Comment (sckott) [reply content/blog/2018-03-20-rentrez-paper.md]: the `https` version of your website seems to have a bad cert 😢

File: content/blog/2018-03-20-rentrez-paper.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+post to find out where papers describing R packages are published these days.
+Although PubMed only covers journals in the biological sciences, searching that
+databse will at least give us an idea of which journals like to publish these
+sorts of papers. Here we use the `entrez_search` and  `entrez_summary` functions 
+to get some  information on all of the papers published in 2017 with the term 
+'R package' in their title:
+
+
+```r
+pkg_search <- entrez_search(db="pubmed",
```
Comment (sckott) [inline content/blog/2018-03-20-rentrez-paper.md]: add `library(rentrez)`?

File: content/blog/2018-03-20-rentrez-paper.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+As you can tell from the output above, you can get a lot of information from
+these summary records. In this case, we are interested in the journals in which
+these papers appear. We can use the helper function `extract_from_esummary`
+to isolate the 'source' of each paper, then use `table` to count up the frequency 
+of each journal.
+
+```r
+journals <- extract_from_esummary(pkg_summs, "source")
+journal_freq <- as.data.frame(table(journals, dnn="journal"), responseName="n.papers")
+ggplot(journal_freq, aes(reorder(journal, n.papers), n.papers)) +
```
Comment (sckott) [inline content/blog/2018-03-20-rentrez-paper.md]: add `library(ggplot2)`?

File: content/blog/2018-03-20-rentrez-paper.md (hunk 4 above)
Comment (dwinter) [reply content/blog/2018-03-20-rentrez-paper.md]: Hmm, certificates sorted out but the https page does not render nicely. Will make sure to play with Jekyll and get it nice before the post goes up on the 20th

**Comment by dwinter** [comment]
Thanks for the feedback @stefaniebutland and @sckott, ~20 comments later and I think we are there.

**Comment by stefaniebutland** [comment]
Thanks @dwinter!! Ready to post on 20th

**Comment by stefaniebutland** [comment]
@dwinter did you notice check failing after your last commit?

**Comment by dwinter** [comment]
Hi @stefaniebutland , 

Yeah.. I had missed Scott's comment about a mistake in the author block. So I

1. Tried to fix it
2. Broke it more comprehensively
3. Reverted to the less-broken version
4. Realized my mistake
5. Properly fixed it

I can only apologize for making such a mess of tiny commits in your history when this is checked in :frowning:

**Comment by stefaniebutland** [comment]
No apologies!! We really appreciate the work you have put into this

**Comment by sckott** [comment]
2nded, thanks for all your work on this @dwinter

**Pull request merged**